server/graphql/vvm/src/mutations/vvm_status_log/insert.rs

```diff
@@ -0,0 +1,87 @@
+use async_graphql::*;
+use graphql_core::standard_graphql_error::StandardGraphqlError::{BadUserInput, InternalError};
+use graphql_core::{standard_graphql_error::validate_auth, ContextExt};
+use repository::vvm_status::vvm_status_log_row::VVMStatusLogRow;
+
+use service::{
+    auth::{Resource, ResourceAccessRequest},
+    vvm::vvm_status_log::insert::{
+        InsertVVMStatusLogError as ServiceError, InsertVVMStatusLogInput as ServiceInput,
+    },
+};
+
+use crate::types::vvm_status_log::VVMStatusLogNode;
+
+#[derive(InputObject)]
+#[graphql(name = "InsertVVMStatusLogInput")]
+pub struct InsertInput {
+    pub id: String,
```

Comment:
We want to allow them to select user_id from a drop list, so need to add here. Date and time can also be selected, so can we add that as an insert input as well (: thanks